tool/templates/run_time_parameters.cfg

```diff
@@ -0,0 +1,5 @@
+# in this file, you can define additional run time parameters which are
+# passed to the java run time when jetty is launched.
+# example: the following line can be uncommented for setting the private
+# key for PersonalTokenService
+# com.siemens.bt.jazz.services.PersonalTokenService.privateKey=abcdefghijklmnop
```

Comment:
Awesome catch! Does this work for all advanced properties (e.g. the whitelist as well)?

Reply:
I've tried and failed to get it working with the whitelist, but that doesn't necessarily mean it can't work, I just haven't figured it out. It works for all settings in the advanced properties though, and not only custom ones.

Reply:
The whitelist is part of the advanced properties, search for  `Jazz Authentication Proxy Whitelist`